### DISCUSSION TITLE
Where is the 'box' to "inventory and organize"?

### DISCUSSION THREAD

**Discussion by joffer77** [sysadminsmedia/homebox#372] [Ideas]
So as I'm checking out homebox, for me it seems the project is missing the 'box'.

What I mean is, where do I put the items I create? I can create (nested) locations, so typically: Home > Garage, Home > Storage Room.
But I wouldn't normally put one item just directly in one of the locations. Most of the time, everything goes into a box on a shelf, which is then stored in the location of choice. That box should get the nicely created QR-code/number-label.

Let's say I have a box with winter shoes. I won't/don't put a QR-code/item-number on each pair of shoes. It won't even stick to the pair, so that's useless. I need to stick the QR-code/number label onto the box. And in that box there will be 10 items of different winter shoes with info and maybe picture(s) (I support the issue about using thumbnails initially when listing items).

With this solution, I can search for (winter) shoes, even use labels, to find the box (#000-001 for instance, which also will tell me where (location) it is stored). If I wonder what's in a specifix box I can check out the box content itself looking up the QR-code/number.

At least this is how I picture homebox or another (home) inventory system would work.

Am I missing something here with homebox? Or does this need to be a feature request?

After thinkering a bit, maybe a workaround for me is to create another "location" which is the storage box itself and just label that one, and use the same Asset ID on all itens in the box ("location"). This seem to work, I just have to manually edit the asset-ID for each item in one box.

Edit 1: Later I found this post, which said it was not a good idea: https://github.com/sysadminsmedia/homebox/discussions/163 (why?)
Edit 2: Digging deeper, I see there is a possibility to put one item as a child of another item (or in general you say which item is the parent of the (new) item)

**Comment by Dbankhansen** [comment]
Indeed... We need a real box/container function

**Comment by tankerkiller125** [comment]
I treat boxes/containers as locations. Given that locations can be children of locations, it works just fine, plus the items tied to said box/container move with it when I change the box/containers parent location.

**Comment by andreyfedoseev** [comment]
I think this can be solved if items are allowed to be used as location. This also goes beyond simple "boxes".

Consider the following example:

- I buy a PC case, which comes in a box.
- I put a bunch of components inside that case, so the case is used as a location for all these items. The case itself is located in my office.
- The branded packaging box the case came in is also an item that I want to track. I case use parent-child relationship to link it the PC case item.
- I fill the box with some books and move it to my garage.

So, I could have the following items:

- PC Case, which serves as location for other items (PC components). It is located in the office.
- PC Case Box, which serves as a location for some books. It is located in the garage. Its parent item is the PC Case.

I could also print a label for the PC Case Box, so it can be located easily in the garage.
